faq.md

```diff
@@ -135,6 +135,12 @@ Setting the default version affects a number of elements including (but not limi
 4. In the files tab, the default version's files would be displayed first.
 5. The docker pull command reflects the default version
 
+## How to run Dockstore with Python 3?
+
+There are currently issues with avro, cwltool, and Python 3.  See [cwltool](https://github.com/common-workflow-language/cwltool/issues/524) for more info.  To work around this issue, instead of installing avro, install avro-cwl.  Therefore, the pip3 requirements.txt file should end up looking like [this](https://raw.githubusercontent.com/ga4gh/dockstore/feature/1220/requirementsFile/dockstore-client/src/main/resources/requirements3.txt)
```

Comment:
Looks like a 404 (probably due to a merge). 
I think this should probably be a link to file served up by a webservice endpoint and shared with Keiran's request